docs/src/ExampleSection.js

```diff
@@ -1,6 +1,7 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import _ from "lodash";
+import moment from "moment";
```

Comment:
why the moment import?

Reply:
@joshbDev So users can use it in the playground section of the docs. I think it's light enough for when people want to experiment with dates on axis.